weblate/templates/translate.html

```diff
@@ -469,7 +469,7 @@ <h5>
             {% if user_can_ignore_check %}
               <a href="{% url 'js-ignore-check' check_id=check.id %}" data-check="{{ check.id }}" data-dismiss-all="{% url 'js-ignore-check-source' check_id=check.id %}" class="btn btn-warning check-dismiss check-dismiss-single">{% trans "Dismiss" %}</a>
               {% if user_can_edit_source %}
-                <label class="dismiss-all"><input type="checkbox" name="dismiss-all-{{ check.id }}" /> {% trans "For all languages" %}</label>
+                <label class="dismiss-all"><input type="checkbox" name="dismiss-all-{{ check.id }}" /> {% trans "For all languages" %}</label><span style="display:inline-block;height:24px;vertical-align:super;margin-left:6px;" title="{% trans "Selecting this option will set the corresponding \"ignore check\" flag" %}">{% icon "info.svg" %}</span>
```

Comment:
Please avoid inline CSS.

Reply:
sure thing, which CSS file would this belong in? The relevant file I can find is https://github.com/WeblateOrg/weblate/blob/main/weblate/static/style-bootstrap.css, is this correct? Or is there a specific stylesheet for the editor?

Reply:
okay I created a class and moved styling to [weblate/static/style-bootstrap.css](https://github.com/WeblateOrg/weblate/blob/main/weblate/static/style-bootstrap.css)